File: Search/Linear Search/Linear Search.java
import java.util.Scanner; // For taking inputs

class Linear_Search
{
	public static void main(String args[])
	{
		Scanner input = new Scanner(System.in);

		int N;
		System.out.print("\n Arrays size : ");
		N = input.nextInt();

		int A[] = new int[N];
		System.out.print("\n Enter elements : ");

		for(int i = 0; i < N; ++i)
			A[i] = input.nextInt();

		int element;
		System.out.print("\n Enter element to be searched : ");
		element = input.nextInt();

		// Linear Search
		//---------------
		for(int i = 0; i < N; ++i)
		{
			if(A[i] == element)
				System.out.println("\n Element " + element + " found at index : " + i);
		}
	}
}

File: Search/Linear Search/Linear Search.py
N = int(input("\n Array size : "))

print("\n Enter elements : ", end="")

A = list(map(int, input().split()))

element = int(input("\n Enter element to be searched : "))

# Linear Search
#---------------
for i in range(N):
	if A[i] == element:
		print("\n Element", element, "found at index :", i)
